flink-tests/src/test/java/org/apache/flink/test/checkpointing/UnalignedCheckpointITCase.java

```diff
@@ -105,7 +103,6 @@
  * </ul>
  */
 @RunWith(Parameterized.class)
-@Category(FailsWithAdaptiveScheduler.class) // FLINK-21689
```

Comment:
Should FLINK-21689 also be added into the issue links of FLINK-34409, since this annotation has been removed?

Reply:
sure